Ginger/Ginger/AutomatePageLib/AddActionMenu/ActionsFactory.cs

```diff
@@ -137,7 +137,7 @@ public static int AddActionsHandler(object mItem, Context mContext, int targetIn
         /// <returns></returns>
         static Act GenerateSelectedAction(Act selectedAction, Context mContext)
         {
-            if (selectedAction.AddActionWizardPage != null)
+            if (selectedAction.AddActionWizardPage != null && !selectedAction.ContainingFolder.Contains("SharedRepository"))
```

Comment:
why not to check check if "selectedAction.IsSharedRepositoryInstance == true"?? never go with validation on strings/names- only when must